types/types.go

```diff
@@ -385,6 +385,11 @@ type SystemContext struct {
 	// Note that this field is used mainly to integrate containers/image into projectatomic/docker
 	// in order to not break any existing docker's integration tests.
 	DockerDisableV1Ping bool
+	// If set, then try to pull layers using bittorent
+	DockerTryTorrent bool
```

Comment:
Do we have to call this Docker?

Reply:
I've called it Docker as it is used by the "docker://" transport as the other options.  Will we keep calling it "docker://"? :-)

Reply:
Yes,  until their is a standard...